settings.gradle

```
@@ -122,6 +122,7 @@ includeWithFlags ':logback13',                           'java', 'publish', 'rel
 project(':logback13').projectDir = file('logback/logback13')
 includeWithFlags ':logback14',                           'java11', 'publish', 'relocate', 'no_aggregation'
 project(':logback14').projectDir = file('logback/logback14')
+includeWithFlags ':micrometer-context',                  'java', 'publish', 'relocate', 'native'
```

@minwoox did you decide to publish this module? If we need some f/u tasks do you drop `publish` flag when this PR is merged?
```suggestion
includeWithFlags ':micrometer-context',                  'java', 'relocate', 'native'
```

---

Alright, let's not publish until we solve the problem. 👍